eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -480,7 +480,7 @@ the main ServiceBusClientBuilder. -->
   <suppress checks="VisibilityModifier" files="com.azure.search.documents.indexes.FieldBuilderTests.java"/>
 
   <!-- Suppression for bug https://github.com/checkstyle/checkstyle/issues/11001 -->
```

Comment:
This issue says the fix is merged are we still seeing the issue?

Reply:
The error still exist.

Reply:
Should we reopen the issue/comment ?

Reply:
Create an issue to track this: https://github.com/Azure/azure-sdk-for-java/issues/31869